AgentBlob/GenerateAgentBlob.ps1

```diff
@@ -0,0 +1,79 @@
+﻿
+#$BootstrapUrl = "http://52.151.39.54/performance"
+
+function Download-7Zip {
+    Write-output "Download-7Zip"
+    $urifile = "https://7-zip.org/a/7z1801-x64.msi"
+    $uri = [System.Uri]$urifile
+    $targetFilename = Split-Path -Path $uri.LocalPath -Leaf
+    write-host $targetFilename
+    Invoke-WebRequest -UseBasicParsing -Uri $urifile -OutFile .\7z1801-x64.msi
```

Comment:
I think we want to download this and install it. Instead of using the current directory for the `OutFile`, please use a file path inside the temporary directory.

Simply create `$targetPath = Join-Path $env:TEMP "7z-x64.msi"` and use this for `-OutFile`